File: desafio_em.cs
// Para ler e escrever dados em C#, utilizamos os seguintes metodos da classe console:
// Console.ReadLine: Lê "UMA" linha com dados de entrada (Input) do usuario;
// Console.writeLine: Imprime um texto de saida (Output) e pulando uma linha;


using System;

public class linguagens_Desafio
{
    public static void Main()
    {
        //Lê os valores de entrada
        float valorSalario = float.Parse(Console.ReadLine());
        float valorBeneficio = float.Parse(Console.ReadLine());

        float valorImposto = 0;
        if (valorSalario>= 0 && valorSalario <= 1100)
        {
        // Atribuiu a aliquota de 5% mediante o salario:
            valorImposto = 0.05f * valorSalario;
        }
        else if (valorSalario >= 1100.01 && valorSalario <= 2500.00){
    valorImposto = 0.10F * valorSalario;
}
 else {
    valorImposto = 0.15F * valorSalario;
 }

// Calcula e imprime o resultado
        float saida = valorSalario - valorImposto + valorBeneficio;
        //Imprime o resultado   
        Console.WriteLine(saida.ToString("0.00"));
        //Console.WriteLine("O valor do salario é: " + valorSalario.ToString("0.00"));
    }

}

File: desafio_em.py
# Para ler e escrever dados em Python, utilizamos as seguintes funções:
# - input: Lê uma linha com dados de entrada do usuário.
# - print: Imprime um texto de saída (Output), pulando linha.


# Função para o cálculo do imposto (baseado nas alíquotas)
def calcular_imposto(salario):
    aliquota = 0.00
    if salario >= 0 and salario <= 1100:
        aliquota = 0.05
    elif salario >= 1100.01 and salario <= 2500.00:
        aliquota = 0.10
    else:
        aliquota = 0.15
    # Calcula o valor do imposto baseado na alíquota e no salário
    return aliquota * salario

# Lê os valores de entrada:
valor_salario = float(input())
valor_beneficio = float(input())

# Calcula o valor do imposto através da função "calcular_imposto".
valor_imposto = calcular_imposto(valor_salario)

# Calcula e imprime a saída (com 2 casas decimais)
saida = valor_salario - valor_imposto + valor_beneficio
print(f"{saida:.2f}")